go/plek.go

```diff
@@ -72,6 +72,12 @@ func New(parentDomain string) Plek {
 // FindURL returns the base URL for the given service name as a *url.URL
 func (p Plek) FindURL(serviceName string) *url.URL {
 	u := &url.URL{Scheme: "https", Host: serviceName + "." + p.parentDomain}
+
+	hostnamePrefix := os.Getenv("PLEK_HOSTNAME_PREFIX")
+	if hostnamePrefix != "" {
+		u.Host = fmt.Sprintf("%s%s", hostnamePrefix, u.Host)
```

Comment:
I nearly did that, but I thought `fmt.Sprintf` might be cleaner. I'm agnostic so happy to change that.